docker-compose.yml

```diff
@@ -1677,6 +1678,18 @@ services:
     volumes: *java-volumes
     command: *java-command
 
+  eclipse-java:
```

Comment:
Should we switch JDK 17 to use eclipse-temurin, too? We can get the latest version of maven that way.

Reply:
> Should we switch JDK 17 to use eclipse-temurin, too? We can get the latest version of maven that way.

+1